python/doc/examples/data_analysis/distribution_fitting/plot_quantilematching_estimator.py

```diff
@@ -0,0 +1,74 @@
+"""
+Define a distribution from quantiles
+====================================
+"""
+
+# %%
+# In this example we are going to estimate a parametric distribution by numerical optimization of some quantiles
+
+# %%
+import openturns as ot
+from openturns.viewer import View
+
+# %%
+# We need as much quantile values as the number of parameters
+# of the distribution, so 2 quantiles for a Gaussian distribution.
```

Comment:
```suggestion
# of the distribution, so 2 quantiles for a normal distribution.
```
With @mbaudin47 @adutfoy and @KieranDelamotte we have been trying to make the terminology homogeneous across the doc: "normal distribution" and "Gaussian vector/process".